src/Symfony/Component/CssSelector/Parser/Handler/StringHandler.php

```diff
@@ -68,7 +68,7 @@ public function handle(Reader $reader, TokenStream $stream)
         }
 
         // check unclosed strings
-        if (strlen($match[0]) === $reader->getRemainingLength()) {
+        if (isset($match[0][$reader->getRemainingLength() - 1])) {
```

Comment:
this is wrong. A single isset cannot check for string length equality (because you have no guarantee that it is the last char of the string)